doc/source/math/scalar.rst

```diff
@@ -68,19 +71,19 @@ Or "What the heck does 'attempted to push NaN onto the stack' mean?".
 kOS does not have a numeric type designed to deal with 
 *imaginary numbers* or *complex numbers*.  Therefore, for
 example, if you attempted to perform ``sqrt(-4)``, you would get
-a "NaN error", rather than the irrational number ``2i``.  NaN means
-"Not a Number" and it means the system is incapable of storing the
+a "NaN error", rather than the irrational number ``2i``.  "NaN" stands
```

Comment:
On re-reading this to review your edit, I noticed a factual error (not merely a formatting or phrasing disagreement) in my original text which needs to be fixed as well:  It claims that ``2i`` is an *irrational* number.  It's not.  It's an *imaginary* number.